pkg/proxy/reverseproxy.go

```diff
@@ -69,18 +71,17 @@ func WithParams(o Params) *Proxy {
 		o.CloseIdleConnsPeriod = DefaultCloseIdleConnsPeriod
 	}
 
-	tr := &http.Transport{
-		Proxy: http.ProxyFromEnvironment,
-		DialContext: (&net.Dialer{
-			Timeout:   30 * time.Second,
-			KeepAlive: 30 * time.Second,
-		}).DialContext,
-		MaxIdleConns:          100,
-		IdleConnTimeout:       90 * time.Second,
-		TLSHandshakeTimeout:   10 * time.Second,
-		ExpectContinueTimeout: 1 * time.Second,
-		MaxIdleConnsPerHost:   o.IdleConnectionsPerHost,
-	}
+	tr := http.DefaultTransport.(*http.Transport)
+	tr.Proxy = http.ProxyFromEnvironment
+	tr.DialContext = (&net.Dialer{
+		Timeout:   30 * time.Second,
```

Comment:
I think it would be nice to have low-level proxy settings configurable with default values set to these. Not a must, but nice to have.